eng/scripts/Track1CodeCheck.ps1

```diff
@@ -91,7 +91,7 @@ try {
     }
     $changeList | ForEach-Object {
         $fileName = $_.filename
-        if ($fileName -match 'sdk/') {
+        if ($fileName -match 'sdk/' -and $fileName -match '/Microsoft') {
```

Comment:
Would this catch data-plane as well? Should it be even more specific to be under Micrsoft.Azure.Management?